react-app/src/components/property/PropertyDetail.tsx

```diff
@@ -351,6 +336,21 @@ const PropertyDetail = (props: IPropertyDetail) => {
             />
           </Box>
         </ParcelMap>
+        {buildingOrParcel === 'Parcel' && (
+          <DataCard
+            loading={propertyLoading}
+            id={'LTSA Information'}
+            values={undefined}
+            title={'LTSA Information'}
+            disableEdit={true}
+            onEdit={undefined}
+          >
+            <TitleOwnership
+              pid={parcel?.parsedBody?.PID ? zeroPadPID(Number(parcel?.parsedBody?.PID)) : null}
+            />{' '}
+            <></>
```

Comment:
Nothing to do with your changes but can you remove this `{' '}<></>`. Seems like a mistake from some other PR.

Reply:
Done.

Reply:
I think that auto-formatting, whether through eslint or not, sometimes does this for weird reasons.